pages/apis/api_differences.md.erb

```diff
@@ -0,0 +1,47 @@
+
+## Features only available in the GraphQL API
+
+* <%= pill "AGENTS", "agents" %> [get a list of agent token IDs (agent tokens are currently only available via GraphQL)](/docs/apis/graphql/graphql-cookbook#get-a-list-of-agent-token-ids).
+* <%= pill "BUILDS", "builds" %> [get all environment variables set on a build](/docs/apis/graphql/graphql-cookbook#get-all-environment-variables-set-on-a-build).
+* <%= pill "BUILDS", "builds" %> [increase the next build number](/docs/apis/graphql/graphql-cookbook#increase-the-next-build-number).
+* <%= pill "BUILDS", "builds" %> [create annotations on a build](/docs/apis/rest-api/annotations).
+* <%= pill "JOBS", "jobs" %> [get all jobs in a given queue for a given timeframe](/docs/apis/graphql/graphql-cookbook#get-all-jobs-in-a-given-queue-for-a-given-timeframe).
+* <%= pill "JOBS", "jobs" %> list job events.
+* <%= pill "ORGANIZATIONS", "organizations" %> [remove users from an organization](/docs/apis/graphql/graphql-cookbook#delete-an-organization-member).
+* <%= pill "ORGANIZATIONS", "organizations" %> manage teams - [add](/docs/apis/graphql/graphql-cookbook#add-an-existing-organization-user-to-a-team) or [remove](/docs/apis/graphql/graphql-cookbook#remove-a-team-member) a team member. Most of the team management features are only available for GraphQL. The REST API can only list teams and can create pipelines in teams.
+* <%= pill "ORGANIZATIONS", "organizations" %> [set up SSO](/docs/integrations/sso/sso-setup-with-graphql).
+* <%= pill "ORGANIZATIONS", "organizations" %> [remove pipeline edit access from existing teams](/docs/apis/graphql/graphql-cookbook#set-teams-pipeline-edit-access-to-read-only-or-build-and-read).
+* <%= pill "PIPELINES", "pipelines" %> [get all the pipeline metrics from the dashboard from the API](/docs/apis/graphql/graphql-cookbook#get-pipeline-metrics).
+* <%= pill "PIPELINES", "pipelines" %> [get the last build's creation date within every pipeline](/docs/apis/graphql/graphql-cookbook#get-the-creation-date-of-the-most-recent-build-in-every-pipeline).
+* <%= pill "PIPELINES", "pipelines" %> [count the number of builds on a branch](/docs/apis/graphql/graphql-cookbook#count-the-number-of-builds-on-a-branch).
+* <%= pill "PIPELINES", "pipelines" %> [get the creation date of the most recent build in every pipeline](/docs/apis/graphql/graphql-cookbook#get-the-creation-date-of-the-most-recent-build-in-every-pipeline).
+* <%= pill "PIPELINES", "pipelines" %> filter results from pipeline listings.
+* <%= pill "PIPELINES", "pipelines" %> create and manage pipeline schedules.
+* <%= pill "USERS", "users" %> [create and revoke](/docs/apis/rest-api/access-token) user API tokens (while REST can only revoke user API tokens).
+* <%= pill "USERS", "users" %> [create a user into a specific team and permissions set](/docs/apis/graphql/graphql-cookbook#create-a-user-add-them-to-a-team-and-set-user-permissions).
```

Comment:
Slight wording thing — organizations don't create users, they invite users.

Organization membership management, is exclusive to GraphQL, including inviting users to become members with specific teams or with a specific role.

Reply:
Thanks for the comment, @sj26! I'll push this update and another update already on my mind to this PR during the EU hours today. Highly appreciated! <3

Reply:
I appreciate all the work you're putting into this! 💚